app/controllers/items_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ItemsController < ApplicationController
+
+  def new
+      @user = User.find(params[:user_id])
+      @item = Item.new
+  end
+
+
+  def create
+      @user = User.find(params[:user_id])
```

Comment:
This would allow any user to create an item associated with another user. Use `@user = current_user` here.